build/media-suite_compile.sh

```diff
@@ -1846,39 +1846,32 @@ if [[ $mpv != "n" ]] && pc_exists libavcodec libavformat libswscale libavfilter;
         do_checkIfExist
     fi
 
-    if ! mpv_disabled vulkan; then
-        # Get the known_good json and get the commit version of the Vulkan-Headers.
-        # It is assumed that the version number has quotes around them (sed)
-        [[ "$jq" != y ]] && do_pacman_install jq
-        _Vulkan_Version=$(
-            ${curl_opts[@]} https://raw.githubusercontent.com/KhronosGroup/Vulkan-ValidationLayers/master/scripts/known_good.json |
-            jq -r '.repos[] | select(.name == "Vulkan-Headers").commit')
-        [[ "$jq" != y ]] && do_pacman_remove jq
-        _check=(vulkan/vulkan.h)
-        if do_vcs "https://github.com/KhronosGroup/Vulkan-Headers.git#tag=${_Vulkan_Version:-v1.1.106}" vulkan-headers; then
+    _check=(libvulkan.a vulkan.pc)
+    if ! mpv_disabled vulkan &&
+        do_vcs "https://github.com/KhronosGroup/Vulkan-Loader.git" vulkan-loader; then
+        _shinchiro_patches="https://raw.githubusercontent.com/shinchiro/mpv-winbuild-cmake/master/packages"
+        do_uninstall "${_check[@]}"
+        do_patch "$_shinchiro_patches/vulkan-0001-cross-compile-static-linking-hacks.patch"
+        create_build_dir
+        log dependencies ../scripts/update_deps.py --no-build
```

Comment:
Better explicitely run it with `/usr/bin/python3`.

Reply:
mingw python3/python2 comes first in path and might not work.

Reply:
wait, it is better to use msys2 python or mingw?

Reply:
Msys2 one.